Readme.md

```diff
@@ -0,0 +1,39 @@
+
+API for:
+
+User adding-
+    METHOD: "POST"
+    Url: "localhost:8010/user/add"    
```

Comment:
no need to declare port number in Readme file because it's depends upon env variable